### ISSUE TITLE
vivado xsim support for xci or xilinx ip

### ISSUE THREAD

**Issue by skokvermon** [olofk/edalize#344]
Hi, 
it seems that vivado xsim doesn't support generating xilinx ip for simulation. 
It is supported for synthesis

Is it a planned feature?
If not I would be willing to help implementing it.

**Comment by Jbalkind** [comment]
I think we'd all love to see that if you're willing to give it a go :)

**Comment by olofk** [comment]
Yes, that is a known issue. Currently, the vivado backend runs in project mode while xsim runs in non-project mode. My proposal is to convert the xsim backend to run in project mode as well. That should bring a number of other features as well. Haven't got the bandwidth to work on it myself so very happy for any help here.
